### ISSUE TITLE
The models does not convergence

### ISSUE THREAD

**Issue by wanmeiziyoulan** [Legion56/Counting-ICCV-DSSINet#13]
Hi, when I train the model with using SHA, SHB or my own datasets, the loss will never show a downward trend and _et_cnt_ also keep empty. I I keep the default parameter settings. 
I would appreciate your reply! Thank you!

**Comment by Song-gusu** [comment]
yeah. when i trained the model on SHA with eopch = 450 , i got the same problem with you. After that i runned nowtest.py and the final mae is very high，it is quite different from the experimental results in this paper
